tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -605,10 +605,10 @@ def test_mixed_precision_bops_utilization(self):
         MixedPrecisionBopsAllWeightsLayersTest(self).run_test()
         MixedPrecisionWeightsOnlyBopsTest(self).run_test()
         MixedPrecisionActivationOnlyBopsTest(self).run_test()
-        MixedPrecisionBopsAndWeightsMemoryUtilizationTest(self).run_test()
-        MixedPrecisionBopsAndActivationMemoryUtilizationTest(self).run_test()
-        MixedPrecisionBopsAndTotalMemoryUtilizationTest(self).run_test()
-        MixedPrecisionBopsWeightsActivationUtilizationTest(self).run_test()
+        # MixedPrecisionBopsAndWeightsMemoryUtilizationTest(self).run_test()
```

Comment:
add a todo regarding bringing back these tests once we solve the issue with combined bops with other ru metrics